src/library.js

```diff
@@ -784,8 +784,20 @@ LibraryManager.library = {
     abort('Assertion failed: ' + (condition ? UTF8ToString(condition) : 'unknown condition') + ', at: ' + [filename ? UTF8ToString(filename) : 'unknown filename', line, func ? UTF8ToString(func) : 'unknown function']);
   },
 
-  terminate__sig: 'vi',
-  terminate: '__cxa_call_unexpected',
+  __cxa_call_unexpected: function(exception) {
+    err('Unexpected exception thrown, this is not properly supported - aborting');
```

Comment:
This does change the behaviour of this function in the case that it the exception stubs are used... you would no longer see the long message with `DISABLE_EXCEPTION_THROWING was set (likely due to ...`  .  Maybe that is good?  But maybe just revert the two library_exceptions files and make the different conscious change to keep this change as NFC?

Sorry to be nit picky...